documentation/modules/auxiliary/scanner/dcerpc/endpoint_mapper.md

```diff
@@ -0,0 +1,130 @@
+## Description
+
+The endpoint_mapper module queries the EndPoint Mapper service of a remote system to determine what services are available. In the information gathering stage, this can provide some very valuable information.
+
+## Verification Steps
+
+1. Do: ```use auxiliary/scanner/dcerpc/endpoint_mapper```
+2. Do: ```set RHOSTS [IP]```
+3. Do: ```set THREADS [number of threads]```
+4. Do: ```run```
+
+In order to run the module, all we need to do is pass it a range of IP addresses, set the THREADS count, and let it go to work.
+
+## Scenarios
+
+**Running the scanner**
+```
+msf > use auxiliary/scanner/dcerpc/endpoint_mapper 
+msf auxiliary(endpoint_mapper) > show options
```

Comment:
since you show setting the options, no need to show a ```show options```.  You can save 10 lines here

Reply:
Done.